package.json

```diff
@@ -137,6 +137,50 @@
           ],
           "default": "on",
           "description": "Refresh memory views when debugger stops"
+        },
+        "memory-inspector.wordsPerGroup": {
+          "type": "number",
+          "enum": [
+            1,
+            2,
+            4,
+            8,
+            16
+          ],
+          "default": 1,
+          "description": "Default words per group"
+        },
+        "memory-inspector.groupsPerRow": {
+          "type": "number",
+          "enum": [
+            1,
+            2,
+            4,
+            8,
+            16,
+            32
+          ],
+          "default": 4,
+          "description": "Default groups per row"
+        },
+        "memory-inspector.variablesVisible": {
+          "type": "boolean",
+          "default": false,
+          "description": "Show variables column?"
+        },
+        "memory-inspector.asciiVisible": {
+          "type": "boolean",
+          "default": false,
+          "description": "Show ASCII column?"
+        },
```

Comment:
These ID's lead to these settings appearing separated from each other in the settings UI. Let's make it `memory-inspector.columns.{ascii, variables}`.

Reply:
I've updated these preference names and all references